libcodechecker/server/api/report_server.py

```diff
@@ -1973,8 +1983,9 @@ def massStoreRun(self, name, tag, version, b64zip, force):
                     store_handler.collect_paths_events(report, file_ids, files)
 
                 LOG.debug("Storing report")
-                report_id = store_handler.addReport(
+                report_id, has_change = store_handler.addReport(
```

Comment:
Maybe instead of returning `has_change` we could pass `report_path_and_events` ?